runtime/mgmt/mgmtinit.c

```diff
@@ -31,3 +33,106 @@ JNI_OnLoad(JavaVM *vm, void *reserved)
 {
 	return JNI_VERSION_1_8;
 }
+
+/* Convert path to platform encoding.
+ *
+ * @return NULL on error, otherwise return converted string
+ */
+static const char*
+translatePathToPlatform(JNIEnv *env, const char* pathUTF)
+{
+	char *result;
+	int32_t bufferLength = 0;
+	int32_t resultLength = 0;
+
+	PORT_ACCESS_FROM_ENV(env);
+
+	bufferLength = 	j9str_convert(J9STR_CODE_MUTF8, J9STR_CODE_PLATFORM_RAW, pathUTF, strlen(pathUTF), NULL, 0);
+
+	/* get the size of the platform string */
+	if (bufferLength < 0 ) {
+		return NULL;  /* error occurred */
+	} else {
+		bufferLength += MAX_STRING_TERMINATOR_LENGTH;
+	}
+
+	result = (char*)j9mem_allocate_memory(bufferLength, OMRMEM_CATEGORY_PORT_LIBRARY);
+	if (NULL == result)  {
+		return NULL;
+	}
+
+	resultLength = j9str_convert(J9STR_CODE_MUTF8, J9STR_CODE_PLATFORM_RAW, pathUTF, strlen(pathUTF), result, bufferLength);
+
+	/* do the conversion */
+	if ((resultLength < 0) ||  (resultLength + MAX_STRING_TERMINATOR_LENGTH != bufferLength)) {
+		j9mem_free_memory(result);
+		return NULL; /* error occurred */
+	} else {
+		/* add terminator */
+		memset(result + resultLength, 0, MAX_STRING_TERMINATOR_LENGTH);
+		return (const char*)result;
+	}
+}
+
+static void throwNativeOutOfMemError(JNIEnv* env) {
+	J9VMThread *currentThread = (J9VMThread *)env;
+	J9JavaVM *vm = currentThread->javaVM;
+	J9InternalVMFunctions *vmFuncs = vm->internalVMFunctions;
+	vmFuncs->internalEnterVMFromJNI(currentThread);
+	vmFuncs->setNativeOutOfMemoryError(currentThread, 0, 0);
+	vmFuncs->internalReleaseVMAccess(currentThread);
+}
+
+/**
+ * Determines whether file path has user permissions only.
+ *
+ * @param path file path
+ * @return Returns false on Windows and true if the file has only user permissions,
+ * false otherwise on other operation systems.
+ */
+JNIEXPORT jboolean JNICALL 
+Java_sun_management_FileSystemImpl_isAccessUserOnly0(JNIEnv *env, jclass c, jstring path)
+{
+	const char* pathUTF;
+	jboolean result = JNI_FALSE;
+
+	PORT_ACCESS_FROM_ENV(env);
+
+	pathUTF = (*env)->GetStringUTFChars(env, path, NULL);
+	if (NULL == pathUTF) {
+error:
+		throwNativeOutOfMemError(env);
+	} else {
+		struct J9FileStat buf;
+		I_32 status;
+		const char* pathConvert;
```

Comment:
Can you assign this an initial value?  If assigned `pathUTF`, then the `#else` code block can be removed